zulipterminal/ui_tools/boxes.py

```diff
@@ -201,16 +201,14 @@ def private_box_view(
             ]
             recipient_info = ", ".join(
                 [
-                    f"{self.model.user_dict[email]['full_name']} <{email}>"
+                    f"""{STATE_ICON[self.model.user_dict.get(email, None).get("status","inactive") if self.model.user_dict.get(email,None) else "inactive"]} {self.model.user_dict[email]['full_name']} <{email}>"""
```

Comment:
The text being added here is validated before any action is performed, such as sending a message, saving a message as a draft, or even cycling out of the recipient header. Only adding an icon here will not bypass the validation check and hence, prevent the user from performing any of the aforementioned actions.

Reply:
@prah23 thanks for pointing that out! Sorry, I thought I fixed this already, but I just realised I forgot to push it. I'll do so as soon as I get access to my computer again. My bad.